core/txpool/txpool.go

```diff
@@ -183,8 +182,10 @@ type Config struct {
 	AccountQueue uint64 // Maximum number of non-executable transaction slots permitted per account
 	GlobalQueue  uint64 // Maximum number of non-executable transaction slots for all accounts
 
-	Lifetime       time.Duration // Maximum amount of time non-executable transaction are queued
-	ReannounceTime time.Duration // Duration for announcing local pending transactions again
+	Lifetime           time.Duration // Maximum amount of time non-executable transaction are queued
+	ReannounceTime     time.Duration // Duration for announcing local pending transactions again
+	ReannounceRemotes  bool          // Wether reannounce remote transactions or not
```

Comment:
Yes, include locals. It means "the remotes are reannounced as well", which won't be never reannounced before. And the locals are always reannounced.

Reply:
It might be better to update the comment of `ReanounceTime`, to remove "local"

```
ReannounceTime     time.Duration // Duration for announcing pending transactions again
```

Reply:
Yes, the comment should be corrected.